src/03_modules.py

```diff
@@ -9,23 +9,36 @@
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
-# YOUR CODE HERE
+print("Number of arguments: ", len(sys.argv))
+print("=== And here there are ===")
+for argument in sys.argv:
+    print(argument)
+print("=== End of the arguments ===")
 
 # Print out the OS platform you're using:
-# YOUR CODE HERE
+if sys.platform.startswith('linux'):
+    print("You using Linux")
+elif sys.platform.startswith('darwin'):
+    print("You using MacOS")
+elif sys.platform.startswith('win32'):
+    print("You using Windows")
+elif sys.platform.startswith('cygwin'):
+    print("You using Windows/Cygwin")
+elif sys.platform.startswith('aix'):
+    print("You using Aix")
 
 # Print out the version of Python you're using:
-# YOUR CODE HERE
+print("Python version is: ", sys.version)
 
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
-# YOUR CODE HERE
+print("The process ID is:", os.getpid())
 
 # Print the current working directory (cwd):
-# YOUR CODE HERE
+print("The working directory:", os.getcwd())
 
 # Print out your machine's login name
-# YOUR CODE HERE
+print("Machine's name:", os.uname()[4])
```

Comment:
They were looking for `os.getlogin()` here, but I see that this pulls the machine identifier out of a 5 item tuple, very cool